code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1668,6 +1668,13 @@
 	G.icon_state = "grabbed1"
 	G.synch()
 
+/mob/living/carbon/human/proc/eyes_shine() /*Used to check if eyes should shine in the dark. Returns the image of the eyes on the layer where they will appear to shine.
+											Eyes need to have significantly high darksight to shine unless the mob has the XRAY vision mutation. Eyes will not shine if they are covered in any way.*/
+	var/obj/item/organ/internal/eyes/eyes = get_int_organ(/obj/item/organ/internal/eyes)
+	var/const/shine_threshold = 6

```

Comment:
This should be a #define, not a constant, and especially not an inline constant.

Reply:
Which is preferable here? A define limited to this scope that is undefined at the end of the proc or one that is put in __DEFINES\mob.dm and left?

Reply:
The latter.